pkgs/applications/virtualization/toolbox/toolbox_glibc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/src/cmd/create.go b/src/cmd/create.go
+index 74e90b1..113ef80 100644
+--- a/src/cmd/create.go
++++ b/src/cmd/create.go
+@@ -207,6 +207,8 @@ func createContainer(container, image, release string, showCommandToEnter bool)
+ 	toolboxPathEnvArg := "TOOLBOX_PATH=" + toolboxPath
+ 	toolboxPathMountArg := toolboxPath + ":/usr/bin/toolbox:ro"
+ 
++	glibcMountArg := "@glibc@:@glibc@:ro"
```

Comment:
Does this need to be an arg or could we do `"--volume", "@glibc@:@glibc@:ro",`?

Reply:
I lean towards it being an arg, just for clarity if the future patch fails to apply, incase RH does anything strange with the variable ordering.